ouroboros-network/src/Ouroboros/Network/Snocket.hs

```diff
@@ -10,6 +10,10 @@ module Ouroboros.Network.Snocket
   , SocketSnocket
   , socketSnocket
   , rawSocketSnocket
+  , ClientSnocket
```

Comment:
I see that you got client from Client in Node-to-client, but I think it is rather confusing name. You introduce the term RemoteConnectionId in a later patch, how about using `Local` as a prefix instead of `Client`?

Reply:
Yes that will be less confusing.

Reply:
Done; I also defined `LocalConnectionId` type alias.